orderedmap.go

```diff
@@ -40,6 +40,15 @@ func (om *OrderedMap) Get(key interface{}) (interface{}, bool) {
 	return nil, false
 }
 
+// GetPair looks for the given key, and returns the pair associated with it,
+// or nil if not found. The boolean it returns says whether the key is present in the map.
```

Comment:
Nit: might be worth adding a sentence explaining what that brings on top of `Get`, maybe something like:

```
The Pair struct can then be used to iterate over the ordered map from that point, either forward or backward.
```

Reply:
Cool, thanks